sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -292,6 +296,13 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
     };
   }
 
+  getLoggingIdentifier(): boolean | undefined{
+    return this.allowLoggingAccountIdentifiers;
+  }
+
+  getTokenCredentialOptions(): TokenCredentialOptions{
```

Comment:
Formatting here.  Also, we should mark these as internal

Reply:
We can mark as internal. It's not exported publicly

Reply:
marked internal